hypothesis-python/tests/cover/test_float_nastiness.py

```diff
@@ -191,3 +196,41 @@ def test_float_free_interval_is_invalid():
     assert float(lo) < lo < hi < float(hi), 'There are no floats in [lo .. hi]'
     with pytest.raises(InvalidArgument):
         st.floats(lo, hi).example()
+
```

Comment:
It would be nice to have some tests for the combination of `width` and `allow_infinity=False` to assert that we never overflow to infinity in those cases (which was after all the original use case)

Reply:
Done